docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -240,6 +240,19 @@ Release
       admin/publish-artifacts
       admin/publish-docs --production
 
+#. Check that the artifacts are set up correctly:
+
+   The following command tests that the client packages can be installed on a number of platforms.
+   This helps to identify any problems with the published artifacts that may not be evident in the regular tests (e.g. S3 permissions or packaging problems).
+
+   .. prompt:: bash (flocker-0.1.2)$
+
+      admin/test-artifacts
```

Comment:
This seems to be solved (I am running the new tests now) with:

`mkvirtualenv --python=/usr/bin/python new_virtualenv`

Then installing the dependencies with:

`env LDFLAGS="-L$(brew --prefix openssl)/lib" CFLAGS="-I$(brew --prefix openssl)/include" pip install -e .[dev]`

The latter is from https://cryptography.io/en/stable/installation/ and is necessary because there are not yet wheels for `cryptography` for the latest Mac OS.